app/components/DrivingInnovationSection/DrivingInnovationSection.tsx

```diff
@@ -0,0 +1,22 @@
+import Image from "next/image";
+import { MainContainer, H2, P } from "components";
+
+import styles from "./drivingInnovationSection.module.scss";
+import imageSrc from "./img/innovationImage.jpg";
+
+export const DrivingInnovationSection = () => {
+  return (
+    <MainContainer>
+      <section className={styles.component}>
+        <H2 className={styles.title}>Driving innovation and growth in finance</H2>
+        <div className={styles.contentWrapper}>
+          <Image className={styles.image} src={imageSrc} alt="Image" />
```

Comment:
These images shouldn't come from src, they should come from contentful CMS, please add images to Contentful